package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "awesome_books",
+  "version": "1.0.0",
+  "description": "\"# awesome_books\"",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fasaldana/awesome_books.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fasaldana/awesome_books/issues"
+  },
+  "homepage": "https://github.com/fasaldana/awesome_books#readme"
```

Comment:
- [ ] Please add a descriptive readme file. You can use this  [template](https://github.com/microverseinc/readme-template/blob/master/README.md)